runtimes/eden/Cargo.toml

```diff
@@ -25,7 +25,7 @@ std = [
   "pallet-membership/std",
   "pallet-multisig/std",
   "pallet-offences/std",
-  "pallet-randomness-collective-flip/std",
+  "pallet-insecure-randomness-collective-flip/std",
```

Comment:
Do we have an alternative to this pallet provided by Parity?

Reply:
@ETeissonniere SRLabs have pointed us to some alternatives (although as a low priority issue) which we will be looking into as the next task. This is just a renaming for the same package at this stage and some system parachains that I have checked are still relying on it.